docs/en-US/component/carousel.md

```diff
@@ -49,6 +59,7 @@ carousel/card
 
 By default, `direction` is `horizontal`. Let carousel be displayed in the vertical direction by setting `direction` to `vertical`.
 
+When type is `card`, `height` will set `width` on the outermost layer, and the value of `width` is equal to the value of `height`
```

Comment:
I think this default behavior is too weird

[demo](https://element-plus.run/#eyJBcHAudnVlIjoiPHRlbXBsYXRlPlxuICA8ZWwtY2Fyb3VzZWxcbiAgICBoZWlnaHQ9XCI0MDBweFwiXG4gICAgZGlyZWN0aW9uPVwidmVydGljYWxcIlxuICAgIHR5cGU9XCJjYXJkXCJcbiAgICA6YXV0b3BsYXk9XCJmYWxzZVwiXG4gICAgICAgICAgICAgICBzdHlsZT1cIndpZHRoOjYwMHB4XCJcbiAgPlxuICAgIDxlbC1jYXJvdXNlbC1pdGVtIHYtZm9yPVwiaXRlbSBpbiA0XCIgOmtleT1cIml0ZW1cIj5cbiAgICAgIDxoMyB0ZXh0PVwiMnhsXCIganVzdGlmeT1cImNlbnRlclwiPnt7IGl0ZW0gfX08L2gzPlxuICAgIDwvZWwtY2Fyb3VzZWwtaXRlbT5cbiAgPC9lbC1jYXJvdXNlbD5cbiAgPGVsLWNhcm91c2VsXG4gICAgaGVpZ2h0PVwiNDAwcHhcIlxuICAgIGRpcmVjdGlvbj1cInZlcnRpY2FsXCJcbiAgICB0eXBlPVwiY2FyZFwiXG4gICAgOmF1dG9wbGF5PVwiZmFsc2VcIlxuICA+XG4gICAgPGVsLWNhcm91c2VsLWl0ZW0gdi1mb3I9XCJpdGVtIGluIDRcIiA6a2V5PVwiaXRlbVwiPlxuICAgICAgPGgzIHRleHQ9XCIyeGxcIiBqdXN0aWZ5PVwiY2VudGVyXCI+e3sgaXRlbSB9fTwvaDM+XG4gICAgPC9lbC1jYXJvdXNlbC1pdGVtPlxuICA8L2VsLWNhcm91c2VsPlxuPC90ZW1wbGF0ZT5cblxuPHN0eWxlIHNjb3BlZD5cbi5lbC1jYXJvdXNlbF9faXRlbSBoMyB7XG4gIGNvbG9yOiAjNDc1NjY5O1xuICBvcGFjaXR5OiAwLjc1O1xuICBsaW5lLWhlaWdodDogMjAwcHg7XG4gIG1hcmdpbjogMDtcbiAgdGV4dC1hbGlnbjogY2VudGVyO1xufVxuXG4uZWwtY2Fyb3VzZWxfX2l0ZW06bnRoLWNoaWxkKDJuKSB7XG4gIGJhY2tncm91bmQtY29sb3I6ICM5OWE5YmY7XG59XG5cbi5lbC1jYXJvdXNlbF9faXRlbTpudGgtY2hpbGQoMm4gKyAxKSB7XG4gIGJhY2tncm91bmQtY29sb3I6ICNkM2RjZTY7XG59XG48L3N0eWxlPlxuIiwiaW1wb3J0X21hcC5qc29uIjoie1xuICBcImltcG9ydHNcIjoge1xuICAgIFwiZWxlbWVudC1wbHVzXCI6IFwiaHR0cHM6Ly9wcmV2aWV3LTEyMzg4LWVsZW1lbnQtcGx1cy5zdXJnZS5zaC9idW5kbGUvaW5kZXguZnVsbC5taW4ubWpzXCIsXG4gICAgXCJlbGVtZW50LXBsdXMvXCI6IFwidW5zdXBwb3J0ZWRcIlxuICB9XG59IiwiX28iOnsic3R5bGVTb3VyY2UiOiJodHRwczovL3ByZXZpZXctMTIzODgtZWxlbWVudC1wbHVzLnN1cmdlLnNoL2J1bmRsZS9pbmRleC5jc3MifX0=)

Reply:
![image](https://user-images.githubusercontent.com/26755049/236129140-3438cf4c-170a-43a0-a38c-1ef5846a9609.png)

if this is the case

For the user's outer layer and redefines the width, the user-defined width should be used.

Reply:
According to my understanding, After setting the `direction="vertical" type="card" height="400px"` of Carousel, it's width and height are always the same.

Users cannot modify this behavior, I feel it's too weird.

Reply:
<img width="1335" alt="image" src="https://user-images.githubusercontent.com/26755049/236716827-13516942-f28e-4e94-aad2-d9d0957d66db.png">

ok For the behavior that can be modified by the user, I think you can directly use rootOffsetWidth to set the width of the Carousel. This behavior is limited to the use of direction="vertical" type="card"

Reply:
![image](https://user-images.githubusercontent.com/26755049/236717061-2a59b8ae-7305-4872-92a9-6bfb86c6f1d5.png)

Reply:
The latest code has been submitted

Reply:
Although I set different width and height for the Carousel component, the width and height of the final component is always the same.

Reply:
In the case of direction="vertical" type="card", I think it must be set to a square, otherwise the Carousel Item will have inconsistent heights to bring deformation

![image](https://github.com/element-plus/element-plus/assets/26755049/8846a34e-ba82-4c46-8030-53b441516f93)

Reply:
@tolking

Reply:
code updated